workers/repositoryactioncounter.py

```diff
@@ -24,8 +24,10 @@ def __init__(self):
         self.add_operation(self._run_counting, POLL_PERIOD_SECONDS)
 
     def _run_counting(self):
+        yesterday = date.today() - timedelta(days=1)
+
         def batch_query():
-            return database.Repository.select()
+            return model.repositoryactioncount.missing_counts_query(yesterday)
```

Comment:
I guess it's fine assume it won't get backed more than one day.

Reply:
Correct